kolibri/utils/tests/test_cli.py

```diff
@@ -168,8 +171,12 @@ def start_mock(port, *args, **kwargs):
         monkeypatch.setattr(server, 'start', start_mock)
 
         test_port = 1234
-        # ENV VARS are always a string
-        os.environ['KOLIBRI_LISTEN_PORT'] = str(test_port)
+
+        os.environ['KOLIBRI_HTTP_PORT'] = str(test_port)
+
+        # force a reload of conf.OPTIONS so the environment variable will be read in
+        from kolibri.utils import conf
+        conf.OPTIONS.update(options.read_options_file(settings.KOLIBRI_HOME))
```

Comment:
Wait, why are you reading KOLIBRI_HOME from settings? Should read it from conf!

(We should also just take it out of the Django settings, but you didn't do that).